Add columns to SHOW DATABASES for autonomous clusters (#1581)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/ql/administration/show-databases-columns.asciidoc

|databaseID|The database unique ID.||{check-mark}|
|serverID|The server instance ID.||{check-mark}|
|address|Instance address in a clustered DBMS. The default for a standalone database is `neo4j://localhost:7687`. |{check-mark}|{check-mark}|
|role|The current role of the database (`primary`, `secondary`, `unknown`).|{check-mark}|{check-mark}|

> |writer|This will be true if this database node accepts writes which will be the case if this node is the leader for this database in a cluster or if this is a standalone instance.|{check-mark}|{check-mark}|

Reviewer: ```suggestion
|writer| `true` for the database node that accepts writes (this node is the leader for this database in a cluster or  this is a standalone instance.|{check-mark}|{check-mark}|
```
